mlflow/gateway/providers/utils.py

```diff
@@ -54,4 +54,60 @@ def rename_payload_keys(payload: Dict[str, Any], mapping: Dict[str, str]) -> Dic
                     key to the new key.
     :return: A new dictionary containing the transformed keys.
     """
-    return {mapping.get(k, k): v for k, v in payload.items()}
+    new_dict = {}
```

Comment:
Why is this conversion needed?

Is this attempting to preserve additional configurations that can be submitted to the model endpoint without requiring mapping within the AI Gateway? 

Is there any downside to letting users directly specify these optional arguments in the format that the provider expects, which is how the existing `rename_payload_keys` function works?

Reply:
this is done so we don't need to write a lot of custom code for remapping keys of the payload for them to be compatible with huggingface tgi endpoints. it also helps us keep the business logic similar to other providers